frontend/src/static/css/style.scss

```diff
@@ -196,7 +196,7 @@ header {
     .tab-content {
       display: flex;
       padding: 1rem 1.5rem;
-      padding-top: 4rem;
+      padding-top: 2rem;
```

Comment:
Instead of change the padding-top to be `2rem`, you can just remove the padding-top property and set the existing padding at line 198 to have a `2rem` top padding.

Reply:
Yes done.